pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -1051,6 +1051,8 @@ let
       EFI_STUB            = yes; # EFI bootloader in the bzImage itself
       EFI_GENERIC_STUB_INITRD_CMDLINE_LOADER =
           whenOlder "6.2" (whenAtLeast "5.8" yes); # initrd kernel parameter for EFI
+       # Generic compression support for EFI payloads
```

Comment:
Can you update this comment to explain why we restrict it to aarch64 for now? Just so someone stumbling over just the file (not the discussion in the PR) understands why it's there.

Reply:
Added a comment. Thanks!